.circleci/config.yml

```diff
@@ -4,19 +4,18 @@ jobs:
     parallelism: 2
     working_directory: ~/wp-e2e-tests
     docker:
-      - image: circleci/node:8.11.0-browsers
+      - image: 'circleci/node:8.11.0-browsers'
```

Comment:
Is this needed for this change or just auto-formatting?

Reply:
yeah, it's just auto-formatting applied after I removed invalid NODE_CONFIG value. And this file was accidentally included in this PR.